aptible-auth.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'aptible-resource', '~> 1.0'
   spec.add_dependency 'gem_config'
+  spec.add_dependency 'multipart-post', '2.1.1'
```

Comment:
Approving, but does this need to be exactly 2.1.1? This will be very constrictive for dependent apps. We should relax if possible.